content/docs/guide.md

```diff
@@ -356,11 +356,17 @@ since we try to use all the latest features.
 
 ## Check that Packit works
 
-With the configuration above, Packit is able only to build your changes and provide them as RPMs.
-Once you create a pull request, Packit builds the code from the
-pull request in Copr repository and reports back Copr build status to the pull request.
-
-An example of Packit pull request:
+With the configuration above, Packit will build your changes and provide them as RPMs.
+Once you create a pull request, Packit will:
+1. Check out the PR.
+2. Merge it with the main branch (this can be configured by the
```

Comment:
I suppose you mean the target branch of the PR, whatever its name is ("main branch" could be misunderstood as the branch whose name is "main", or the default branch of the repo, like `main`, `master` `devel`, whatever).
The difference shows up when you open a PR against another branch than the one you usually merge to.

Reply:
Yes, Pavel is right. We merge to the target branch.

Reply:
For comparison, here's how Zuul documents its procedure (it uses "target branches"). https://zuul-ci.org/docs/zuul/5.2.2/job-content.html#git-repositories